specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1164,6 +1236,402 @@
         }
       }
     },
+    "potentialOutagesGetResult": {
+      "description": "The Get PotentialOutages operation response.",
+      "type": "object",
+      "$ref": "#/definitions/potentialoutage"
+    },
+    "potentialoutage": {
+      "type": "object",
+      "description": "Potential outages detected by Brain.",
+      "properties": {
+        "id": {
+          "type": "integer",
+          "description": "Id for the potential outage.",
+          "format": "int64"
+        },
+        "title": {
+          "type": "string",
+          "description": "Title for the potential outage."
+        },
+        "summary": {
+          "type": "string",
+          "description": "Summary for the potential outage."
+        },
+        "confidence": {
+          "type": "number",
+          "description": "Confidence score for the potential outage."
+        },
+        "displayConfidence": {
+          "type": "string",
+          "description": "Display confidence score for the potential outage."
+        },
+        "plane": {
```

Comment:
If the values are know, this could be an enum. Again not required now, but would be good.

Reply:
Yeah I agree. Should be modeled as an enum.

---
In reply to: [552320999](https://github.com/Azure/azure-rest-api-specs/pull/12206#discussion_r552320999) [](ancestors = 552320999)

Reply:
As of now, i'd like to keep as it is, and will address it if I got more certain information on this field.

Reply:
ok

---
In reply to: [552703443](https://github.com/Azure/azure-rest-api-specs/pull/12206#discussion_r552703443) [](ancestors = 552703443)